app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3:
+        raise ValueError("Invalid command format. "
+                         "Please provide the command in the format:"
+                         " cp <source_file> <destination_file>")
+
+    _, source_file, test_file = parts
+
+    if source_file == test_file or _ != "cp":
```

Comment:
In case you use variable value, do not omit is since `_` just skips value and is not intended for storing
```suggestion
    command_name, source_file, test_file = parts

    if source_file == test_file or command_name != "cp":
```